csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,106 +5,88 @@ namespace ToDo
-                // Show current taks
-                for (int i = 0; i < TL.Count; i++)
-                {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
-                }
-                Console.WriteLine("----------------------------------------");
+            Console.WriteLine("Enter the name of the task: ");
+            string task = Console.ReadLine();
 
-                string line = Console.ReadLine();
-                // Remove one position
-                int indexToRemove = Convert.ToInt32(line) - 1;
-                if (indexToRemove > -1)
-                {
-                    if (TL.Count > 0)
-                    {
-                        string task = TL[indexToRemove];
-                        TL.RemoveAt(indexToRemove);
-                        Console.WriteLine("Task " + task + " deleted");
-                    }
-                }
-            }
-            catch (Exception)
-            {
-            }
+            Tasks.Add(task);
+
+            Console.WriteLine("Task registered successfully");
         }
 
-        public static void ShowMenuAdd()
+        public static void RemoveTask()
         {
-            try
-            {
-                Console.WriteLine("Enter the name of the task: ");
-                string task = Console.ReadLine();
-                TL.Add(task);
-                Console.WriteLine("Task registered successfully");
-            }
-            catch (Exception)
+            Console.WriteLine("Enter the number of the task to remove: ");
+
+            ShowCurrentTasks();
+
+            int indexToRemove = Convert.ToInt32(Console.ReadLine()) - 1;
+
+            // Validate and remove one position
+            if (indexToRemove > -1)
             {
+                if (Tasks.Count > 0)
+                {
+                    string task = Tasks[indexToRemove];
+                    Tasks.RemoveAt(indexToRemove);
+                    Console.WriteLine("Task " + task + " deleted");
+                }
             }
         }
 
-        public static void ShowMenuTres()
+        private static void ShowCurrentTasks()
         {
-            if (TL == null || TL.Count == 0)
+            Console.WriteLine("----------------------------------------");
+
+            if (Tasks == null || Tasks.Count == 0)
             {
                 Console.WriteLine("There are no tasks to perform");
-            } 
-            else
+            }
+
+            for (int index = 0; index < Tasks.Count; index++)
             {
-                Console.WriteLine("----------------------------------------");
-                for (int i = 0; i < TL.Count; i++)
-                {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
-                }
-                Console.WriteLine("----------------------------------------");
+                Console.WriteLine((index + 1) + ". " + Tasks[index]);
             }
```

Comment:
Instead of using a `for` loop, you can use the `foreach` statement, it's more natural for reading and we will be following the KISS principle. Also, you should use string interpolation to have better logic to build the message. What do you think about refactoring this logic like this:
```suggestion
            var indexTask = 1;
            TaskList.ForEach(task => Console.WriteLine($"{indexTask++}. {task}"));```